Assets/Scripts/NPC/NPCBehavior.cs

```diff
@@ -215,5 +288,16 @@ public void triggerInfectedSoundsInInterval()
                 SetTimeInterval();
             }
         }
+
+        /// <summary>
+        /// For debug purposes.
+        /// </summary>
+        public void SetBehaviors(bool isNoMask, bool isNoVac, MaskType maskType)
```

Comment:
Is that intended for debug only?

Reply:
it's used later for the settings. Will change the comment on future commit